src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="main_conteiner main_conteiner--month-length--29">
+        <div class="main_calendar main_calendar--start-day--sun"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
+        <div class="main_calendar"></div>
```

Comment:
make calendar separate BEM block

Reply:
I hope I understood the problem correctly...